js/g_spot.js

```diff
@@ -336,9 +339,14 @@ export default async function init(config) {
 
                 event.target.setStyle(layerConfig.styleHighlight);
                 fLayer.bringToFront();
+
+                chartManager.renderCharts(f.properties);
+                lfSidebarControl.open("stats");
+                addDistrictNameToQuestionnaireHeader(f.properties.obns_cyr);
```

Comment:
Would you mind if we add a new functions called

```
const showStats = (feature) => {
    chartManager.renderCharts(feature.properties);
    lfSidebarControl.open("stats");
    addDistrictNameToQuestionnaireHeader(feature.properties["obns_cyr"]);
};
const hideStats = () => {
    lfSidebarControl.close();
};
```

This way I can call you at any time?